snorkel/contrib/candidate_viewer.py

```diff
@@ -0,0 +1,128 @@
+import sqlite3
+
+
+class CandidateViewer:
```

Comment:
can you add a doctoring with init parameters and example usage here?

Reply:
I added a docstring and example usage, but please let me know if the format is not correct.